drivers/metal/rendering_device_driver_metal.mm

```diff
@@ -3947,6 +3956,8 @@ bool isArrayTexture(MTLTextureType p_type) {
 			return false;
 		case SUPPORTS_FRAGMENT_SHADER_WITH_ONLY_SIDE_EFFECTS:
 			return true;
+		case SUPPORTS_BUFFER_ADDRESS:
+			return false;
```

Comment:
This can be true, but I would like to implement it by adding a new boolean property on the `MetalFeatures` struct:

https://github.com/godotengine/godot/blob/f27764ecec09469f13b25716f1adef59f1e63821/drivers/metal/metal_device_properties.h#L73